python/sglang/srt/two_batch_overlap.py

```diff
@@ -415,9 +468,72 @@ def prepare_raw(
             out_num_token_non_padded=out_num_token_non_padded_b,
         )
 
+        if is_enable_two_chunk:
+            cls.filter_batch_for_two_chunk(
+                batch,
+                child_a=child_a,
+                child_b=child_b,
+                tbo_split_seq_index=batch.tbo_split_seq_index,
+            )
+
         assert batch.tbo_children is None
         batch.tbo_children = [child_a, child_b]
 
+    @classmethod
```

Comment:
seems this function can be short:

1. properly set child_a.extend_seq_lens_cpu, child_b.extend_seq_lens_cpu, etc
2. call that "derive all other fields" util